py3status/modules/zypper_updates.py

```diff
@@ -0,0 +1,70 @@
+"""
+Display number of pending updates for OpenSUSE Linux.
+
+Display the system updates counted by zypper lu.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds
+        (default 600)
+    format: Display format to use
+        (default 'zypper: [\?color=update {update}]')
+
+Format placeholders:
+    {updates} number of pending zypper updates
+
+Color options:
+    color_bad: out to date system
+    color_degraded: Upgrade available
+    color_good: No upgrades needed
+
+@author Ioannis Bonatakis <ybonatakis@suse.com>
+@license BSD
+
+SAMPLE OUTPUT
+{'full_text': 'zypper: 137', 'color': '#FFFF00', 'cached_until': 1616323657.0}
```

Comment:
Default `format` is:

```suggestion
[{'full_text': 'zypper: '}, {'full_text': '0', 'color': '#00FF00'}]
```

Reply:
i have something like `[{'full_text':·'zypper:·',·'separator':·False,·'separator_block_width':·0,·'cached_until':·1616348096.0},·{'full_text\|        keys = list(mod_config)                                                                                       
':·'0',·'color':·'#00FF00'}]` . But i just followed your suggestion